made a page AboutMe

NosatskyiK opened the pull request:
Added title text, main text and main image on page about me. Added mobile a version

Acceptance criteria for 1

- [x]  Mentor is added to pr
- [x]  AC copied to pr description
- [x]  link to deployed version added to pr
- [x]  About block (text and image) added to the main page
- [x]  On the widths 390px and 1280px, the site layout corresponds to the design

Acceptance criteria for 1*

- [x]  Created separate component About
- [x]  Its styles added to [.module.css](https://github.com/css-modules/css-modules)
- [ ]  For App.tsx styles moved to App.module.css


P.S. Маю певні сумніви щодо правильності виконання останнього пункту про перенос стилів до App.module.css, я створив окремі css файл для компоненту і всі стилі знаходяться там, а в App.css прописав тільки декілька обнуляючих стилів. Якщо зробив все таки не правильно, то можу пофіксити якщо трішки поясните цей пункт. Дякую)

Link - https://nosatskyik.github.io/fe-react-2024/

Commit:
made a page AboutMe

Commit:
added mobile a version

vadym-zinchenko-moc commented on src/index.css:
```diff
@@ -1,71 +1,71 @@
-:root {
```
why you commented styles? if you don't use this maybe need to delete?

vadym-zinchenko-moc commented on src/components/AboutMe/aboutMe.css:
```diff
@@ -0,0 +1,74 @@
+  font-weight: inherit;
+}
+
+@media (max-width: 390px) {
```
the mobile version does not match the design
![Screenshot 2024-04-21 at 22 08 12](https://github.com/NosatskyiK/fe-react-2024/assets/115080771/79a8db18-9471-42bb-9ff4-1998b3f44dc5)

Commit:
resized for mobile and added style modules

NosatskyiK commented:
Resized for mobile devices and added style modules, also removed unnecessary styles from main.css file

vadym-zinchenko-moc commented on src/App.tsx:
```diff
@@ -1,19 +1,11 @@
 
 function App() {
     return (
         <>
```
this is extra Fragment, because you have only one element inside 
`return (
   <AboutMe />
)`

vadym-zinchenko-moc reviewed (changes requested):
<img width="661" alt="Screenshot 2024-04-22 at 20 33 14" src="https://github.com/NosatskyiK/fe-react-2024/assets/115080771/4a5ad2ea-58a0-4a3d-89e5-b9e0a65c79f7">
<img width="512" alt="Screenshot 2024-04-22 at 20 33 24" src="https://github.com/NosatskyiK/fe-react-2024/assets/115080771/63d376b8-01bb-4cd9-a941-e9de33c7e4d5">

I don't see that issue is resolved with media requests.

vadym-zinchenko-moc commented:
<img width="310" alt="Screenshot 2024-04-22 at 20 31 31" src="https://github.com/NosatskyiK/fe-react-2024/assets/115080771/64cf2fd1-6973-4ecf-bc1a-77468c23c818">

why file App.css inside folder app. What the reason for this? this file must be near App component

vadym-zinchenko-moc commented:
<img width="309" alt="Screenshot 2024-04-22 at 20 43 53" src="https://github.com/NosatskyiK/fe-react-2024/assets/115080771/8b7cd6e5-e11a-4132-af95-132672ef1755">

folder images must be inside assets

vadym-zinchenko-moc commented:
<img width="1112" alt="Screenshot 2024-04-22 at 20 43 40" src="https://github.com/NosatskyiK/fe-react-2024/assets/115080771/266a636f-1278-443d-b5e7-451235ea5624">

naming must be in one style (if `header` then must be `aboutMe` )

Commit:
fixed bugs

NosatskyiK commented:
1. Added adaptability, now when the browser window is reduced, the image adapts and becomes smaller
2. Deleted the unnecessary fragment in App.tsx
3. Moved the app.css file to App.tsx
4. Moved the folder with images to the assets folder
5. Renamed the aboutMe file

vadym-zinchenko-moc commented on src/components/aboutMe/aboutMe.module.css:
```diff
@@ -0,0 +1,99 @@
+    margin: 10px;
+  }
+
+  .img {
```
it's very bad naming for classes `img` `txt `...you have tag `img`. Maybe you can use something more understandable `aboutImage` for example

vadym-zinchenko-moc replied:
and other classes must be more particular

vadym-zinchenko-moc commented on src/components/aboutMe/aboutMe.module.css:
```diff
@@ -0,0 +1,99 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap");
+
+body {
```
you already added styles for body in App.css file. Why do you use againg. Can you do this in one file?

vadym-zinchenko-moc commented on src/components/aboutMe/AboutMe.component.tsx:
```diff
@@ -0,0 +1,42 @@
+
+import styles from './aboutMe.module.css';
+
+export const AboutMe = function () {
```
you can use arrow functions.

vadym-zinchenko-moc commented on src/components/aboutMe/AboutMe.component.tsx:
```diff
@@ -0,0 +1,42 @@
+import title from '@/assets/img/title.png';
```
is it `title`? isn't `about_image` maybe for example?

vadym-zinchenko-moc replied:
are you can name `car`  for example like `bed` for example?? you need to name everything understandable. classes, images etc...

vadym-zinchenko-moc commented on src/components/aboutMe/aboutMe.module.css:
```diff
@@ -0,0 +1,99 @@
+
+  .img {
+    height: auto;
+    width: 80%;
```
<img width="315" alt="Screenshot 2024-04-23 at 11 42 55" src="https://github.com/NosatskyiK/fe-react-2024/assets/115080771/6c18e6f8-b505-4f91-9717-83b49667fe37">

in design we image and text in one line.

Commit:
fixed bugs

NosatskyiK commented:
1. Changed the names of classes
2. Moved common styles to app.css
3. Replaced the normal function with an arrow function in AboutMe.component.tsx
4. Changed the main image
5. In the mobile version, align text and images on one line
